test/sanity/validate-modules/main.py

```diff
@@ -1082,19 +1084,29 @@ def _validate_docs(self):
 
         return doc_info, doc
 
-    def _check_version_added(self, doc):
-        if not self._is_new_module():
-            return
-
+    def _check_version_added(self, doc, existing_doc):
+        version_added_raw = doc.get('version_added')
         try:
             version_added = StrictVersion(str(doc.get('version_added', '0.0') or '0.0'))
         except ValueError:
             version_added = doc.get('version_added', '0.0')
+            if version_added != 'historical':
+                self.reporter.error(
+                    path=self.object_path,
+                    code=306,
+                    msg='version_added is not a valid version number: %r' % version_added
+                )
+                return
+
+        existing_version = existing_doc.get('version_added')
```

Comment:
This will fail on new modules since `existing_doc` will be `None`.

Reply:
Resolved